src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,8 @@
+@keyframes clock-hand-rotate {
+  from {
+    transform: rotate(-180deg);
+  }
+  to {
+    transform: rotate(180deg);
```

Comment:
You can use here turn value for transform: rotate([rotate MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate)) for simplicity.